flow_test.go

```diff
@@ -514,9 +514,9 @@ func BenchmarkFlowUnmarshal(b *testing.B) {
 
 	for n := 0; n < b.N; n++ {
 		// Make a new copy of the AD to avoid reinstantiation.
-		iad := ad
+		iad := *ad
```

Comment:
Woops. This would be a good case for `netlink.AttributeDecoder.Reset()`. Would you be willing to contribute that to Matt's library?

Reply:
I personally do not think an `AttributeDecoder` should be reused at all, which means I do not think `Reset` is a valuable addition to the public API of `AttributeDecoder`. It only makes sense here in the context of a benchmark, taking into account the fact that using `StartTimer` / `StopTimer` would not be ideal here and would make the benchmark slower.